tests/Feature/MovieCreateTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+
+class MovieCreateTest extends TestCase
+{
+    use RefreshDatabase;
+    /**
+     * A basic feature test example.
+     *
+     * @return void
+     */
+    public function test_movie_create_screen_requires_authorized_user()
+    {
+        $user = User::factory()->make();
+
+        $this->actingAs($user)
+        ->get('/movies')
+        ->assertStatus(200);
```

Comment:
assertOk() can be used instead of assertStatus(200). It just makes things more readable.